doc/Cifar10.rst

```diff
@@ -0,0 +1,1108 @@
+Estimating prediction sets on the Cifar10 dataset
+=================================================
+
+What is done in this tutorial ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   -  **Cifar10 dataset** : 10 classes (horse, dog, cat, frog, deer,
+      bird, airplane, truck, ship, automobile)
+
+..
+
+   -  Use :class:``mapie.classification.MapieClassifier`` to compare the
```

Comment:
Only one ` is needed before and after mapie.classification.MapieClassifier.

Reply:
done